docs/serving/using-a-tls-cert.md

```diff
@@ -253,16 +251,17 @@ Kubernetes secret and then configure the `knative-ingress-gateway`:
               number: 80
               protocol: HTTP
           - hosts:
-              - "*"
+              - TLS_HOSTS
             port:
               name: https
               number: 443
               protocol: HTTPS
             tls:
               mode: SIMPLE
-              privateKey: /etc/istio/ingressgateway-certs/tls.key
-              serverCertificate: /etc/istio/ingressgateway-certs/tls.crt
+              credentialName: tls-cert
       ```
+      In above example, `TLS_HOSTS` represents the hosts of your TLS certificate. It can be single host, multiple hosts, or wildcard host.
```

Comment:
```suggestion
      In the example above, `TLS_HOSTS` represents the hosts of your TLS certificate. It can be a single host, multiple hosts, or a wildcard host.
```